src/app/base/components/CertificateFields/CertificateFields.test.tsx

```diff
@@ -16,12 +16,17 @@ describe("CertificateFields", () => {
         />
       </Formik>
     );
-    expect(wrapper.find("textarea[name='certificate']").exists()).toBe(false);
-    expect(wrapper.find("textarea[name='key']").exists()).toBe(false);
+
+    expect(
+      screen.queryByRole("textbox", { name: "Upload certificate" })
```

Comment:
Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.Could use an enum here too.

I think my keyboard got sick of my pasting this everywhere so it just pasted it a million times for some reason.